tox.ini

```diff
@@ -195,6 +200,8 @@ commands_pre =
 
   asgi: pip install {toxinidir}/ext/opentelemetry-ext-asgi
 
+  asyncpg: pip install {toxinidir}/ext/opentelemetry-ext-asyncpg
+
   boto: pip install {toxinidir}/ext/opentelemetry-ext-boto[test]
 
   flask: pip install {toxinidir}/ext/opentelemetry-ext-flask[test]
```

Comment:
I think you need `asyncpg` in the `lint` environment as well.

Reply:
it's already there or i didn't understand something